demos/demos.js

```diff
@@ -1,4 +1,4 @@
-import React from 'react'
+import React from 'preact-compat'
 import ReactDOM from 'react-dom'
```

Comment:
I know this isn't required for `n-ui`, but shall we move to `preact-dom`?

Reply:
Otherwise 💯

Reply:
ah they shoudl both be preact-compat. I missed that one